### PR TITLE
User and role, grid and detail

### PR DESCRIPTION
**Pull request by cmangeat**
User grid and detail

### CODE REVIEW

**Comment by sbrunner** [comment]
I don't know what exactly you want to say with acceptance test, in c2cgeomapfish we put unit tests in the folder `tests`, and functional tests in the folder `tests/functional`... than is there any reason to add a new concept based on the meaning of acceptance?

**Comment by arnaud-morvan** [comment]
Can you give a title related with the content of your PR.

**Comment by cmangeat** [comment]
I took the name name from wsgiutils. 
As this point, there are many levels tests in this dir, grouped by functionalities, "acceptance" meaning that the test file related functionality meet requirements.
We can also choose to allocate same functionality related tests in many dirs.
In my opinion, theses tests are not unit tests, plenty of them targeting dev from c2cgeoform or common.
Learn tests can also be seen as canaries.
I couldn't choose tests for name as this name is already used for test instrumentation (importable) at common side.

File: admin/Makefile [hunk 1]
```diff
@@ -43,7 +43,8 @@ ${BUILD_DIR}/requirements.timestamp: ${BUILD_DIR}/venv.timestamp ../commons/setu
 	${VENV}/bin/pip install -e "."
 	touch $@
 
-${BUILD_DIR}/requirements-dev.timestamp: ${BUILD_DIR}/venv.timestamp setup.py
+${BUILD_DIR}/requirements-dev.timestamp: ${BUILD_DIR}/venv.timestamp requirements.txt setup.py
```
Comment (arnaud-morvan) [inline admin/Makefile]: requirements.txt should be in ${BUILD_DIR}/requirements.timestamp not in ${BUILD_DIR}/requirements-**dev**.timestamp

**Comment by arnaud-morvan** [comment]
@sbrunner : I've rebase on master, and cleaned code. Can you have a last look, I'd like to merge this ASAP.

**Pull request merged**